lib/push.js

```diff
@@ -253,190 +253,148 @@ module.exports = function (log, db, config) {
 
   return {
 
-    isValidPublicKey: isValidPublicKey,
+    isValidPublicKey,
 
     /**
-     * Notifies all devices that there was an update to the account
+     * Notify devices that there was an update to the account
      *
      * @param {String} uid
+     * @param {Device[]} devices
      * @param {String} reason
+     * @param {Object} [options]
+     *   @param {String[]} [options.includedDeviceIds]
+     *   @param {String[]} [options.excludedDeviceIds]
+     *   @param {String} [options.data]
+     *   @param {String} [options.TTL] (in seconds)
      * @promise
      */
-    notifyUpdate: function notifyUpdate(uid, reason) {
-      reason = reason || 'accountVerify'
```

Comment:
This default value for `reason` was never being used outside of the tests. Redundant complexity so I killed it.